runtime/stout/src/lib.rs

```diff
@@ -353,7 +352,7 @@ pub type AssetBalance = Balance;
 impl pallet_assets::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type Balance = AssetBalance;
-	type AssetId = AssetId;
+	type AssetId = AssetIdForTrustBackedAssets;
```

Comment:
> In case we wanted to add assets from an untrusted source we could take some inspiration from cumulus where there's 2 instances of pallet_assets (Trusted and ForeignAssets)
You are wright on that, the main reason is to leave the door open  to implement this in the future but this is open to discussion.